requirements.txt

```diff
@@ -0,0 +1,11 @@
+loguru==0.5.3
+bioc==2.0.post1
+numpy==1.21.2
+# pandas==1.3.3
+pybrat==0.1.4
+datasets==2.0.0
+black~=22.0
+flake8>=3.8.3
+isort>=5.0.0
+aiohttp==3.8.1
+openpyxl>=3.0.9
```

Comment:
we have a `setup.cfg` file that should handle this, were you having import errors?

Reply:
Nope, just an artifact. It is now deleted.